client/blocks/reader-combined-card/index.jsx

```diff
@@ -33,9 +33,22 @@ class ReaderCombinedCard extends React.Component {
 	}
 
 	componentDidMount() {
-		const { posts, site, feed } = this.props;
-		const feedId = get( feed, 'feed_ID' );
-		const siteId = get( site, 'ID' );
+		this.recordRenderTrack();
+	}
+
+	componentDidUpdate( nextProps ) {
+		if ( size( this.props.posts ) !== size( nextProps.posts ) ) {
+			this.recordRenderTrack();
+		}
+	}
+
+	recordRenderTrack = () => {
+		const { posts } = this.props;
+
+		// Use site ID/feed ID from the first post, because we may not have site and feed objects yet
+		const siteId = posts[ 0 ].blog_ID;
```

Comment:
it's the thing that we turn the array of post keys into, which includes things with isCombined and isRecommendationBlock. the stuff in the items array in the reader/stream